src/status_im2/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -0,0 +1,196 @@
+      :subtitle        "3-5 min"}]
+    [quo/data-item
+     {:container-style {:flex   1
+                        :height 36}
+      :blur?           false
+      :description     :default
+      :icon-right?     false
+      :card?           false
+      :label           :none
+      :status          :default
+      :size            :small
+      :title           (i18n/label :t/max-fees)
+      :subtitle        "€188,70"}]
+    [quo/data-item
+     {:container-style {:flex   1
+                        :height 36}
+      :blur?           false
+      :description     :default
+      :icon-right?     false
+      :card?           false
+      :label           :none
+      :status          :default
+      :size            :small
+      :title           (i18n/label :t/user-gets {:name "Mark"})
+      :subtitle        "149.99 ETH"}]]])
+
+(defn- f-view-internal
+  [_]
+  (let [reset-slider?        (reagent/atom false)
+        margin-top           (safe-area/get-top)
+        biometric-auth?      true
+        on-close             #(rf/dispatch [:navigate-back-within-stack :wallet-select-asset])
+        status-account-props {:customization-color :purple
+                              :size                32
+                              :emoji               "🍑"
+                              :type                :default
+                              :name                "Collectibles vault"
+                              :address             "0x0ah...78b"}
+        user-props           {:full-name           "M L"
+                              :status-indicator?   false
+                              :size                :small
+                              :ring-background     (resources/get-mock-image :ring)
+                              :customization-color :blue
+                              :name                "Mark Libot"
+                              :address             "0x0ah...78b"
+                              :status-account      (merge status-account-props
+                                                          {:size  16
+                                                           :name  "New house"
+                                                           :emoji "🍔"})}]
+    (fn [{:keys [theme]}]
+      [rn/view {:style {:flex 1}}
+       [quo/gradient-cover {:customization-color :purple}]
+       [rn/view {:style (style/container margin-top)}
+        [quo/page-nav
+         {:icon-name           :i/arrow-left
+          :on-press            on-close
+          :accessibility-label :top-bar
+          :right-side          [{:icon-name           :i/advanced
+                                 :on-press            (fn callback [] nil)
+                                 :accessibility-label "Advanced"}]}]
+        [transaction-title]
+        [transaction-from status-account-props theme]
+        [transaction-to user-props theme]
+        [transaction-details theme]
+        [rn/view {:style style/slide-button-container}
```

Comment:
a nit here, we should instead use `container-style` prop on the actual component because it is much easier to read imo.

also here we should be using `slide-button` from status-im2.common.standard-authentication.core instead as this will handle the auth flows 👍 - 
It's a follow up anyway and the UI will be the same 👍

Reply:
Yes, I replaced on the integration with backend PR. Will merge this first and replace it shortly with the integration part ✨